core/debugger/remote_debugger.cpp

```diff
@@ -529,6 +530,26 @@ void RemoteDebugger::debug(bool p_can_continue, bool p_is_error_breakpoint) {
 			} else if (command == "set_skip_breakpoints") {
 				ERR_FAIL_COND(data.is_empty());
 				script_debugger->set_skip_breakpoints(data[0]);
+			} else if (command == "evaluate") {
```

Comment:
One last thing, do we want this to work in release builds? We usually disable some of the more "introspective" debugger features in release mode. As an example, the live tree editing only works in debug builds.

In that case, we probably want to protect this else statement with a:

```
#ifdef DEBUG_ENABLED
// [code]
#endif
```

Reply:
Does RemoteDebugger even exist in release builds? That would make no sense.